### DISCUSSION TITLE
No me aparece práctica HTML

### DISCUSSION THREAD

**Discussion by PatricioNovellinoLaunchx** [Launch-X-Latam/MisionFrontEnd#282] [Q&A] [answered]
Hola que tal a todos disculpen por ser domingo pero es el día que me puedo poner a hacer.
En el apartado HTML aparece que esta la sección práctica, pero luego cuando entro, no me aparece. Y tampoco en el navegador.
No se si el MC habrá dicho algo en las últimas clases, me falta ver las últimas 2 aún porque tenía entendido que era para entregar el 4/11

![Screenshot_20221030-193547_GitHub.png](https://user-images.githubusercontent.com/114259588/198905126-2d208db2-fc16-4688-8fb4-95949f5b3e8e.png)
![Screenshot_20221030-193605_GitHub.png](https://user-images.githubusercontent.com/114259588/198905130-e6614f13-811a-4d80-82e4-c30748fba56c.png)

**Comment by Cerecero** [comment, marked as answer]
Es [esta](https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/02%20-%20HTML/practicas). Se muestra en donde vez todas las carpetas de HTML.

**Reply by PatricioNovellinoLaunchx** [reply]
Aahhh. Muchisimas gracias! Todavía me vengo amigando con github no sabía que se podía buscar por carpetas también

**Comment by doguedogue** [comment]
Fijate en la ruta de los directorios
![image](https://user-images.githubusercontent.com/23409026/198908600-89a263f2-da90-4ef4-8de4-b8ba3ccd146f.png)
